modular_shadow/code/modules/events/electrical_storm.dm

```diff
@@ -0,0 +1,2 @@
+/datum/round_event_control/electrical_storm
+	max_occurrences = 2
```

Comment:
```suggestion
	max_occurrences = 3
```
I agree that a weight of 40 with no max_occurrences is dumb. But 2 is too few. 4 would be too many, so 3 is the middle ground here.

Reply:
There, it's three now